src/layout/Body.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
It feels like this file's component is `page` rather reusable `layout`? I'd expect layout to contain `children` or some sort of logic to render subsequent components without any logic. Means layout typically is dumb component

Reply:
definition of dumb component: https://medium.com/@thejasonfile/dumb-components-and-smart-components-e7b33a698d43

Reply:
yes, I was messing up with page and layout,
I will create fixed version of layout folder with Navigation, Footer components and Layout component like you mentioned.